coq-mathcomp-algebra-tactics.opam

```diff
@@ -21,7 +21,7 @@ depends: [
   "coq" {(>= "8.13" & < "8.14~") | (= "dev")}
   "coq-mathcomp-algebra" {(>= "1.12" & < "1.13~") | (= "dev")}
   "coq-mathcomp-zify" {(>= "1.0.0") | (= "dev")}
-  "coq-elpi" {(>= "1.10.0") | (= "dev")}
+  "coq-elpi" {(>= "1.10.1") | (= "dev")}
```

Comment:
@gares Could you edit `meta.yml` and regenerate files (excluding `.github/workflows/docker-action.yml`) instead?